packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -168,6 +197,17 @@ export interface EditableTree extends Iterable<EditableField> {
      * and for `sequence` fields the content ends up in order of "sequenced-last" to "sequenced-first".
      */
     [createField](fieldKey: FieldKey, newContent: ITreeCursor | ITreeCursor[]): void;
+
+    /**
+     * Replaces the field of this node.
```

Comment:
It would be good to clarify the merge semantics of the current code:
- For value and optional fields, any concurent edit are overwritten in a last-write-wins fashion.
- For sequence fields, the nodes present in the field on the issuing client will be deleted and the `newContent` will be inserted. This means concurrent inserts (including calls to `replaceField`) can all contribute content. In the future this will likely be replaced with merge semantics that is more consistent with that of optional and value fields.

Reply:
I like this wording. Fixed.